typings.json

```diff
@@ -1,6 +1,8 @@
 {
-  "ambientDependencies": {
+  "globalDependencies": {
     "es6-shim": "registry:dt/es6-shim#0.31.2+20160317120654",
-    "jasmine": "registry:dt/jasmine#2.2.0+20160412134438"
+    "core-js": "registry:dt/core-js#0.0.0+20160602141332",
```

Comment:
Seems like one of either `es6-shim` or `core-js` is needed, but not both.  right?